roles/nfd_operator_deploy_custom_commit/files/operator_image_builder_pod.yml

```diff
@@ -0,0 +1,46 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    app: nfd-operator-ci
+  name: operator-image-builder-pod
+  namespace: nfd-operator-ci
+spec:
+  containers:
+  - command:
+    - /bin/sh
+    - /mnt/helper/run_operator_image_builder.sh
+    image: image-registry.openshift-image-registry.svc:5000/nfd-operator-ci-utils/nfd-operator-ci-utils:builder_helper
+    name: operator-image-builder
+    imagePullPolicy: Always
+    securityContext:
+      privileged: true
+    env:
+      - name: OPERATOR_GIT_REPO
+        value: "{{ nfd_operator_git_repo }}"
+      - name: OPERATOR_GIT_REF
+        value: "{{ nfd_operator_git_ref }}"
+      - name: IMAGESTREAM_NAME
```

Comment:
what about `IMAGESTREAM_NAMESPACED_NAME`, to avoid the confusion I had? 
(`namespaced name` is k8s [terminology](https://pkg.go.dev/k8s.io/apimachinery/pkg/types#NamespacedName))